src/cljs/commiteth/common.cljs

```diff
@@ -2,7 +2,7 @@
   (:require [reagent.core :as r]
             [re-frame.core :as rf]
             [clojure.string :as str]
-            [cljsjs.moment]))
+            [goog.date.relative :as relative]))
```

Comment:
When using a namespace only a few times I often prefer to not alias it, i.e.
```
[goog.date.relative]
```
This makes it easier to read the code without keeping all the aliases in your head.

This is general feedback, you don't need to change this to get this PR merged. 👍

Reply:
that feedback makes sense! just updated without the alias for the sake of consistency